frontend/src/main/java/com/google/tripmeout/frontend/storage/TripStorage.java

```diff
@@ -12,7 +12,7 @@
    *
    * @param trip the TripModel object to add to storage
    *
-   * @throws a TripAlreadyExists exception if there is a TripModel object 
+   * @throws a TripAlreadyExists exception if there is a TripModel object
```

Comment:
FYI, having run the formatter both here and in #31 means you'll have to deal with a merge conflict on submission. Should be easy to deal with, but I wanted to point that out.